objects/_o-link-complex.scss

```diff
@@ -69,26 +71,14 @@ $o-link-complex-apply-at-breakpoints-for-target:   false !default;
 
   %o-link-complex__target,
   .o-link-complex__target {
-    color: $link-color;
     cursor: pointer;
-
-    &:hover,
-    &:focus {
-      text-decoration: underline;
-      color: $link-color-hover;
-    }
+    @include base-link();
```

Comment:
Me and others (https://css-tricks.com/sass-style-guide/, http://www.sitepoint.com/css-sass-styleguide/) are fans of including mixins (excluding rem/prefix mixins) before any normal properties, so this would be:

```scss
%o-link-complex__target,
.o-link-complex__target {
  @include base-link();
  cursor: pointer;
}
```